pygfx/renderers/wgpu/engine/blender.py

```diff
@@ -364,14 +364,14 @@ def get_color_attachments(self, pass_type):
                 "name": "accum",
                 "resolve_target": None,
                 "clear_value": accum_clear_value,
-                "load_op": wgpu.LoadOp.clear,
+                "load_op": wgpu.LoadOp.load,
```

Comment:
Hey @almarklein i hope this 2 line change is OK. 

it allows one to keep these buffers alive between calls to `renderer.render` so that things can blend well!

You can see the streamed validation example for the effect.

Reply:
I just came by a usecase where being able to "clear" between calls to `render` is useful. not sure if can find a way to control this behavior